src/main/java/InfrastructureManager/ModuleManagement/ModuleDebugInput.java

```diff
@@ -48,10 +48,14 @@ public String read() throws InterruptedException {
 
     /**
      * Store a debug level message in the logging queue
+     * @param name Name of the entity
      * @param message Message to be stored
      */
-    public void debug(String message){
-        this.store("DEBUG - " + message);
+    public void debug(String name, String message){
+        //this.store("DEBUG - " + message);
+        //String[] temp = message.split("-",2);
+        //temp[0]=temp[0].replaceAll("\\s","");
+        this.store("DEBUG-"+name+"-"+ Thread.currentThread().getStackTrace()[2].getMethodName()+":"+Thread.currentThread().getStackTrace()[2].getLineNumber() +" -" + message);
```

Comment:
there is always a space before and after operators (a + b  not a+b), and there is a single space after a comma but not before (a, b not a,b).
apply for all of the changes (should be a setting in your IDE, called auto format or something)